stable/datadog/templates/containers-common-env.yaml

```diff
@@ -22,4 +22,12 @@
 {{- end }}
 - name: KUBERNETES
   value: "yes"
+{{- if .Values.datadog.site }}
+- name: DD_SITE
+  value: {{ .Values.datadog.site | quote }}
+{{- end }}
+{{- if .Values.datadog.dd_url }}
+- name: DD_DD_URL
+  value: {{ .Values.datadog.dd_url | quote }}
+{{- end }}
```

Comment:
if you add those environments variables into the common env, you should also remove them from the container definitions, like here: https://github.com/helm/charts/blob/29272e429f0ffa5868e4381c23e7a0ba108201e9/stable/datadog/templates/container-agent.yaml#L22

Reply:
I have removed the duplicate references of DD_SITE and DD_DD_URL